lib/aws/rails/middleware/elastic_beanstalk_sqsd.rb

```diff
@@ -31,37 +31,42 @@ def call(env)
         private
 
         def execute_job(request)
-          # Jobs queued from the Active Job SQS adapter contain the JSON message in the request body.
-          job = Aws::Json.load(request.body.string)
+          # Jobs queued from the SQS adapter contain the JSON message in the request body.
+          job = ::ActiveSupport::JSON.decode(request.body.string)
           job_name = job['job_class']
           @logger.debug("Executing job: #{job_name}")
-
-          begin
-            ::ActiveJob::Base.execute(job)
-          rescue NameError => e
-            @logger.error("Job #{job_name} could not resolve to a class that inherits from Active Job.")
-            @logger.error("Error: #{e}")
-            return internal_error_response
-          end
-
+          _execute_job(job, job_name)
           [200, { 'Content-Type' => 'text/plain' }, ["Successfully ran job #{job_name}."]]
```

Comment:
Nit, if we have `internal_error_response` should we also have a method for successful respones?

Reply:
The messages are different, there is not much re-use there.